core/src/message/common.rs

```diff
@@ -24,7 +24,10 @@ use codec::{Decode, Encode};
 use core::ops::Deref;
 use scale_info::TypeInfo;
 
-/// Message.
+/// Message 
+/// 
+/// It is a structure for message between actors. 
```

Comment:
No need to write `It is` here.

```suggestion
/// Structure for a message between actors. 
```

Reply:
done